src/test/java/ua/biz/test/SQLDataProv/EmployeeSQLDataProvider.java

```diff
@@ -0,0 +1,56 @@
+package ua.biz.test.SQLDataProv;
+
+import org.testng.annotations.Test;
+import ua.biz.test.entity.Vacancy;
+import ua.biz.test.enums.EmployeeColumnName;
+import ua.biz.test.enums.TableName;
+import ua.biz.test.entity.Employee;
+import ua.biz.test.utils.credentials.DBCredHelper;
+
+import java.sql.*;
+
+public class EmployeeSQLDataProvider {
+
+    public static void setEmployeeIdForEmployee(Employee employee) {
```

Comment:
didn't get the reason to set id if u have emp_number

Reply:
The method has a wrong name. Because I thought the unique user code is id not the number.